Curtains/Curtains.Application/CurtainsService/MessageConstructor.cs

```diff
@@ -10,6 +10,7 @@ namespace Curtains.Application.CurtainsService
     /// </summary>
     class MessageConstructor : IMessageConstructor
     {
+        private const string _customerName = "Раиса Леонидовна";
```

Comment:
Config?
Today customer Raisa in another day Nik

Reply:
done